lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -444,11 +461,20 @@ def create_patient(record: REDCapRecord) -> tuple:
         'preferred': True # Assumes that the project language is the patient's preferred language
     }]
 
-    patient_id = generate_patient_hash(
-        names       = (record['participant_first_name'], record['participant_last_name']),
-        gender      = gender,
-        birth_date  = record['birthday'],
-        postal_code = record['home_zipcode_2'])
+    # Use the UW NetID to generate patient id if it's available.
+    # This is the stable identifier that can allow us to match UW reopening
+    # participants.
+    #   -Jover, 04 September 2020
+    net_id = record.get('netid')
+    net_id = net_id.strip() if net_id else None
+    if net_id:
+        patient_id = generate_hash(net_id)
```

Comment:
Let's normalize case too with `net_id.strip().lower()`?

Reply:
Aha, yup. Thank you!